core/experiments/manager.php

```diff
@@ -504,7 +525,20 @@ private function render_feature_settings_field( array $feature ) {
 				<div class="e-experiment__status"><?php echo sprintf( esc_html__( 'Status: %s', 'elementor' ), $this->release_statuses[ $feature['release_status'] ] );  // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></div>
 			<?php } ?>
 		</div>
-		<?php
+		<?php $this->render_feature_dependency( $feature );
+	}
+
+	private function render_feature_dependency( $feature ) {
+		if ( ! empty( $feature['dependency'] ) ) {
+			?>
+			<div class="e-experiment__dependency">
+				<b class="e-experiment__dependency__title"><?php echo esc_html__( 'Depends on:', 'elementor' ); ?></b>
+			<?php foreach ( $feature['dependency'] as $dependency ) : ?>
+				<span class="e-experiment__dependency__item"><?php echo $dependency->get_name(); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?></span>
```

Comment:
escaping is probably unnecessary, but it might be a better practice to escape anyway instead of turning the PHPCS off

Reply:
Handled.